server/monitor/src/main/java/org/apache/accumulo/monitor/servlets/BasicServlet.java

```diff
@@ -264,4 +264,12 @@ protected static void banner(StringBuilder sb, String klass, String text) {
     sb.append("<br />\n<h2 class='").append(klass).append("'>").append(text).append("</h2>\n");
   }
 
+  /**
+   * Creates a {@link Cookie} with the given name and value, also setting the HttpOnly attribute on the cookie.
+   */
+  protected static Cookie createCookie(String name, String value) {
+    Cookie c = new Cookie(name, value);
+    c.setHttpOnly(true);
```

Comment:
> Are we certain we're not manipulating them in client-side javascript now? If we are, this will break that.

I can double-check, but I'm not aware of that in 1.x. Not sure how the table sorting stuff works. It would be nice to quell these warnings in the future, but I may begrudgingly have to revert.